content/applications/general/in_app_purchase.rst

```diff
@@ -2,58 +2,63 @@
-By default, IAP accounts are common to all companies, but can be restricted to specific
-ones. Activate the :ref:`developer mode <developer-mode>`, then go to :menuselection:`Technical
-Settings --> IAP Account`.
+Credits for IAP services are stored on IAP accounts, which are specific to each service. By default,
+IAP accounts are common to all companies, but can be restricted to specific ones. To make changes to
+IAP accounts, activate Odoo’s :ref:`developer mode <developer-mode>`. Return to the main apps page
+and type *IAP*. In the resulting dropdown menu select
+:guilabel:`Settings / Technical / IAP / IAP Accounts`.
 
 .. image:: in_app_purchase/image2.png
    :align: center
+   :alt: The IAP Account menu showing IAP services, companies, and account tokens.
 
 .. tip::
-   An IAP account can be disabled by appending `+disabled` to its token.
-   Reverting this change will re-enable the account.
+   To disable an IAP account, append `+disabled` to its token. To re-enable the account, remove
+   `+disabled` from the token.
 
-IAP Portal
+IAP portal
 ==========
 
-The IAP Portal is a platform regrouping your IAP Services. It is accessible from
-:menuselection:`Settings app --> Odoo IAP --> View my Services`. From there, you can view your current
-balance, recharge your credits and set a reminder when your balance falls below a threshold.
+The IAP portal is a platform for managing all IAP services. Actions that can be performed in the
+portal include checking an account's balance, buying more credits (recharge), and setting reminders
+to be automatically notified when an account’s balance falls below a specified threshold. To access,
+the IAP portal, go to the :menuselection:`Settings app --> Search "IAP" --> View My Services`.
 
 .. image:: in_app_purchase/image3.png
    :align: center
+   :alt: The Your Services page showing selected services, current balance, and the Recharge button.
 
 Get notified when credits are low
 =================================
 
-To be notified when it’s time to recharge your credits, you can go to your IAP Portal through
-:menuselection:`Settings app --> Odoo IAP --> View my Services`, unfold a service and check the
-Receive threshold warning option. Then, you can provide a minimum amount of credits and email
-addresses. Now, every time that the limit is reached, an automatic reminder will be sent by
-email!
+To be notified when credits are low, go to the
+:menuselection:`Settings app --> Search "IAP" --> View My Services`. Unfold a service and check the
+:guilabel:`Receive threshold warning` box. Enter a minimum credit amount in the
+:guilabel:`Warning threshold` field. Finally, click :guilabel:`Change Email` and provide an email
+address to be automatically notified when credits are at or below the :guilabel:`Warning threshold`.
 
 .. image:: in_app_purchase/image4.png
    :align: center
+   :alt: The Receive threshold warning box is checked and a Warning threshold is set.
```

Comment:
May want to add a section on manually editing the token in Odoo and where to edit (dev mode --> settings app--> technical menu-->
IAP-->
IAP Accounts
OR
IAP Partners

Reply:
agreed, added this to the IAP Accounts section